core/src/main/java/org/infinispan/factories/DistributionManagerFactory.java

```diff
@@ -9,7 +9,7 @@
    @Override
    @SuppressWarnings("unchecked")
    public <T> T construct(Class<T> componentType) {
-      if (configuration.clustering().cacheMode().isDistributed() || configuration.clustering().cacheMode().isReplicated())
+      if (configuration.clustering().cacheMode().isClustered())
```

Comment:
My final goal is to use `DistributionManager` for all ownership related stuff, regardless of clustering mode (although local caches may remain without one), so I can remove `ClusteringDependentLogic` completely.